compiler/luci-interpreter/src/kernels/CMakeLists.txt

```diff
@@ -64,6 +64,8 @@ set(SOURCES
     Reverse.cpp
     Rsqrt.h
     Rsqrt.cpp
+    Sub.cpp
```

Comment:
Please keep alphabetical order (in other places too).
It is OK to fix incorrect ordering if you find one.

Reply:
Done

Reply:
I mean, Squeeze < Sub < Tanh

Reply:
Yes of course.